scripts/common.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#! /usr/bin/env bash
```

Comment:
Yah dropped the -e. Any reason?

Reply:
Copied it from somewhere! Can you give me the full, correct, reference I should use? I'll normalize it everywhere.

Reply:
Reading up on it, I'm not sure we want to use the -e. We set error handling at the top of each script and they don't all use the same behavior. Thoughts?

Reply:
Yeah you can't give more than one parameter in a shebang according to shellcheck. If you want error checking when using '/usr/bin/env bash' then you have to set it with a 'set'